darts/models/forecasting/block_rnn_model.py

```diff
@@ -30,6 +30,7 @@ def __init__(
         nr_params: int,
         num_layers_out_fc: Optional[List] = None,
         dropout: float = 0.0,
+        activation: Optional[str] = None,
```

Comment:
This is not optional anymore. Can you update it everywhere, and also adjust the parameter description by removing the "Optionally, "?
```suggestion
        activation: str = "ReLU",
```